ui/app/templates/components/config-pki-ca.hbs

```diff
@@ -9,7 +9,32 @@
   </h2>
   {{#if (or model.certificate model.csr)}}
     {{#each model.attrs as |attr|}}
-      {{info-table-row data-test-table-row label=(capitalize (or attr.options.label (humanize (dasherize attr.name)))) value=(get model attr.name)}}
+      {{#if (or 
+        (eq attr.name "csr")
+        (eq attr.name "certificate")
+        (eq attr.name "issuingCa")
+        (eq attr.name "caChain")
+        (eq attr.name "privateKey")
+      )}}
+        <InfoTableRow @label={{capitalize (or attr.options.label (humanize (dasherize attr.name)))}} @value={{get model attr.name}}>
+          <MaskedInput
+            @value={{get model attr.name}}
+            @displayOnly={{true}}
+          />
+        </InfoTableRow>
+      {{else if (eq attr.name "expiration")}}
```

Comment:
While I was in here, I made it so the expiration was human readable. See Expiration row.
![image](https://user-images.githubusercontent.com/6618863/115052766-90b78d80-9e9b-11eb-8168-05a082d726cd.png)